### ISSUE TITLE
documentation of generics for roleExperiments

### ISSUE THREAD

**Issue by diazrenata** [role-model/roleR#101]
1. in general this is confusing me
2. https://stackoverflow.com/questions/66225058/document-new-s4-method-for-an-existing-generic
3. the existing ones all reference `experimentMeta` which no longer exists. 
https://stackoverflow.com/questions/4396768/how-to-properly-document-s4-and-methods-using-roxygen

**Comment by diazrenata** [comment]
o m g this is v precise

```

#' Extract parts of a roleExperiment
#' @name [
#' @aliases [,roleExperiment-method
#' @docType methods
#' @rdname roleExperiment

setMethod("[", 
         signature(x = "roleExperiment", i = "ANY", j = "ANY"),
          function(x, i, j, ..., drop = FALSE) {
```

key appears to be:

- need the @ name field
- need @ aliases and there cannot be a space between the comma and the roleExperiment-method part

**Comment by diazrenata** [comment]
solved in commit 0463e12
